hudi-cli/src/main/resources/log4j2.properties

```diff
@@ -25,7 +25,7 @@ appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = %-4r [%t] %-5p %c %x - %m%n
 
 # Root logger level
-rootLogger.level = warn
+rootLogger.level = info
```

Comment:
Should this be kept as `warn`, since we already have explicit rules below to print `info`-level logs for `org.apache.hudi.common`, `org.apache.hudi.cli`, and `org.apache.spark`?